### DISCUSSION TITLE
Fleetctl add query

### DISCUSSION THREAD

**Discussion by ernstdieter** [Security-Onion-Solutions/securityonion#7310] [Unsupported Versions]
Hi guys,
I read the documentation and I did all the steps to add queries to Fleet, but the number is still the same, nothing happens.

docker exec -it so-fleet fleetctl config set --address https://localhost:8080 --url-prefix fleet --tls-skip-verify

[+] Set the address config key to "https://localhost:8080" in the "default" context
[+] Set the tls-skip-verify config key to "true" in the "default" context
[+] Set the url-prefix config key to "fleet" in the "default" context

docker exec -it so-fleet fleetctl login
Log in using the standard Fleet credentials.
Email: *******@*****.....
Password:
[+] Fleet login successful and context configured!

docker exec -it so-fleet fleetctl apply -f /packs/my-perf.yaml
[+] applied 4 queries
[+] applied 1 packs

-> But nothing changed. I have a distributed environment, fleet is on a seperate host. 
I did this from SO Master system and from Fleet Host. From command line it looks fine, but in Fleet nothing changed. I did this with many packages from Palantir, but nothing appears in Fleet.

Any idea what's wrong?

I'm using SO 2.3.91

Best Regards
Ernst

**Comment by defensivedepth** [comment]
Hello there!

Can you please post a sanitized output of the following?


Run from the Manager:

`sudo salt-call pillar.get global`
